variables.tf

```diff
@@ -8,6 +8,11 @@ variable "cidr_block" {
   description = "The CIDR block for the VPC"
 }
 
+variable "context" {
```

Comment:
Ok, so this becomes `name` instead. Sounds fine to me and think we don't even update the description. Still seems to fit and be a nice way to describe it IMO.

Reply:
Change applied